File: hola_mundo/src/main/java/com/example/hola_mundo/controllers/HomeController.java
package com.example.hola_mundo.controllers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HomeController {
    @GetMapping("/")
    public String inicio(){
        return "<h1>Página de inicio</h1>";
    }

    @GetMapping("/saludos")
    public String saludos(){
        return "<h2>Saludos, Atlacomulco</h2>";
    }
}

File: hola_mundo_kotlin/src/main/kotlin/com/example/hola_mundo_kotlin/controllers/HomeController.kt
package com.example.hola_mundo_kotlin.controllers

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HomeController {
    @GetMapping("/")
    fun inicio(): String{
        return "<h1>Página de inicio</h1>"
    }
}